vendor/idrive_vnc.js

```diff
@@ -36,4 +38,48 @@ function handleConnect() {
   if ( myParam !== null && myParam.length > 0) {
     document.getElementById('status').textContent = "Connected to - " + myParam;
   }
+
+  // Handle Clipboard Paste
+  let clipboardButton = document.getElementById('sendClipboardButton');
+  clipboardButton.addEventListener("click", function() {
+    if (!window.isSecureContext || null === myRFB) {
+      return;
+    }
+
+    navigator.permissions.query({name: 'clipboard-read'}).then(result => {
+      if (result.state == "granted" || result.state == "prompt") {
+        navigator.clipboard.readText().then(text => {
+          if (text.length > 0) {
+            sendClipBoardTextToConsole(text);
+          }
+        });
+      }
+    });
+  });
+
+  function sendClipBoardTextToConsole(text) {
```

Comment:
Where is this code from?

Reply:
Can you not use [this API](https://novnc.com/noVNC/docs/API.html#rfbclipboardpastefrom) ?

Reply:
I have tried this but this didn't work as expected